docs/src/modules/components/AppLayoutDocsFooter.js

```diff
@@ -389,19 +436,31 @@ export default function AppLayoutDocsFooter() {
                 margin="dense"
                 name="comment"
                 fullWidth
-                rows={6}
+                rows={4}
                 value={comment}
                 onChange={handleChangeTextfield}
                 inputProps={{
-                  'aria-label': t('feedbackCommentLabel'),
                   'aria-describedby': 'feedback-description',
```

Comment:
It feels more like a label than a description

```suggestion
                  'aria-labelledby': 'feedback-description',
```

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby